apis/supernova/src/handler/hypercube-handler.js

```diff
@@ -124,12 +165,12 @@ class HyperCubeHandler extends DataPropertyHandler {
   addMeasure(measure, alternative, idx) {
     const meas = initializeField(measure);
 
-    if (hcHelper.isMeasureAlternative(this, meas, alternative)) {
+    if (hcUtils.isMeasureAlternative(this, meas, alternative)) {
       const hcMeasures = this.hcProperties.qLayoutExclude.qHyperCubeDef.qMeasures;
-      return hcHelper.addAlternativeMeasure(meas, hcMeasures, idx);
+      return hcUtils.addAlternativeMeasure(meas, hcMeasures, idx).then(() => measure);
```

Comment:
Don't think then is needed here since addAlternativeMeasure already returns a resolved promise with measure